Dockerfile.debug

```diff
@@ -147,13 +147,12 @@ EXPOSE 2020
 # Default: Run under valgrind
 # alternatively, you can change this to run FB directly and try
 # to get a core file instead
-CMD valgrind --leak-check=full --error-limit=no /fluent-bit/bin/fluent-bit -c /fluent-bit/etc/fluent-bit.conf
+# CMD valgrind --leak-check=full --error-limit=no /fluent-bit/bin/fluent-bit -c /fluent-bit/etc/fluent-bit.conf
 
 # Alternate: Run directly and get core file
 # make sure you run with `docker run --ulimit core=-1`
 # and mount the /cores folder onto the host
 # Un-Comment the lines below:
 # 
-# RUN mkdir /cores && chmod 777 /cores
-# CMD /fluent-bit/bin/fluent-bit -c /fluent-bit/etc/fluent-bit.conf
- 
+RUN mkdir /cores && chmod 777 /cores
+CMD /fluent-bit/bin/fluent-bit -c /fluent-bit/etc/fluent-bit.conf
```

Comment:
Why change the default for the debug dockerfile from valgrind to core option?

Reply:
You're right. I don't like this.